src/cascadia/WinRTUtils/WtExeUtils.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "WtExeUtils.h"
+
+// Method Description:
+// - Escapes the given string so that it can be used as a command-line arg within a quoted string.
+// - e.g. given `";foo\` will return `\"\;foo\\` so that the caller can construct a command-line
+//   using something such as `fmt::format(L"wt --title \"{}\"", EscapeCommandlineArg(TabTitle()))`.
+// Arguments:
+// - arg - the command-line argument to escape.
+// Return Value:
+// - the escaped command-line argument.
+std::wstring EscapeCommandlineArg(const std::wstring_view arg)

```

Comment:
I'm pretty sure that the `_TIL_PREFIXINLINE` that is used for the other methods in the _WtExeUtils.h_ file is so they can be header-only and not run into ODR failures. If you follow that pattern and put this code in the header, I don't think you need the CPP or to change the VCXPROJ file.

Reply:
Yep, that works - done.